query-node/mappings/src/content/channel.ts

```diff
@@ -3,13 +3,47 @@ eslint-disable @typescript-eslint/naming-convention
 */
 import { EventContext, StoreContext } from '@joystream/hydra-common'
 import { Content } from '../../generated/types'
-import { convertContentActorToChannelOrNftOwner, processChannelMetadata, unsetAssetRelations } from './utils'
-import { Channel, ChannelCategory, StorageDataObject, Membership } from 'query-node/dist/model'
-import { deserializeMetadata, inconsistentState, logger } from '../common'
-import { ChannelCategoryMetadata, ChannelMetadata } from '@joystream/metadata-protobuf'
+import {
+  convertContentActor,
+  convertContentActorToChannelOrNftOwner,
+  processChannelMetadata,
+  unsetAssetRelations,
+} from './utils'
+import {
+  Channel,
+  ChannelCategory,
+  StorageDataObject,
+  Membership,
+  MetaprotocolTransactionStatusEvent,
+  MetaprotocolTransactionPending,
+  MetaprotocolTransactionErrored,
+  MetaprotocolTransactionSuccessful,
+} from 'query-node/dist/model'
+import {
+  deserializeMetadata,
+  genericEventFields,
+  inconsistentState,
+  invalidMetadata,
+  logger,
+  updateMetaprotocolTransactionStatus,
+} from '../common'
+import {
+  ChannelCategoryMetadata,
+  ChannelMetadata,
+  ChannelModeratorRemarked,
+  ChannelOwnerRemarked,
+} from '@joystream/metadata-protobuf'
 import { integrateMeta } from '@joystream/metadata-protobuf/utils'
 import { In } from 'typeorm'
+import {
+  processModerateCommentMessage,
+  processPinOrUnpinCommentMessage,
+  processBanOrUnbanMemberFromChannelMessage,
+  processCommentSectionPreferenceMessage,
+  processVideoReactionsPreferenceMessage,
+} from './commentAndReaction'
 import { getAllManagers } from '../derivedPropertiesManager/applications'
+import { BaseModel } from '@joystream/warthog'
 
 export async function content_ChannelCreated(ctx: EventContext & StoreContext): Promise<void> {
   const { store, event } = ctx
```

Comment:
```suggestion
export async function content_ChannelCreated({ store, event }: EventContext & StoreContext): Promise<void> {
```
Let's keep the new code consistent with existing mappings. Please fix it in all other places as well.